sys/net/gnrc/network_layer/sixlowpan/iphc/gnrc_sixlowpan_iphc.c

```diff
@@ -852,6 +853,7 @@ void gnrc_sixlowpan_iphc_recv(gnrc_pktsnip_t *sixlo, void *rbuf_ptr,
                 }
             }
             if ((ipv6 == NULL) || (res < 0)) {
+                /* TODO fall-back to reassembly in minfwd when ipv6 != NULL */
```

Comment:
Can you elaborate?

Reply:
It might be, that just a return would be enough here to trigger the mentioned reassembly, need to check.

Reply:
Ah no `ipv6` is changed for sending... I will try to make the comment clearer.